src/components/Pager.vue

```diff
@@ -0,0 +1,374 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2024 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+<template>
+  <div class="pager">
+    <template v-if="pages.length === 1">
+      <slot name="page" :page="pages[0]" />
+    </template>
+    <template v-else>
+      <div class="container">
+        <Gutter class="left">
+          <ControlPrevious
+            :disabled="!hasPreviousPage"
+            @click.native="previous"
+          />
+        </Gutter>
+        <div class="viewport" ref="viewport">
+          <div
```

Comment:
Pages that are not visible should be hidden, not only visually but from all users (including screen reader users). We should add a class with  `display: none` or the HTML `hidden` attribute.

Reply:
The structure is purposefully set up to use overflow and visually hiding the overflowing pages to allow for natural scrolling/swiping. Using something like `display: none` would eliminate the functionality of scrolling between pages since we wouldn't actually be using overflow anymore in that scenario.

Reply:
Yep, I understand the concern. Let's leave it as it is then!